src/Plugin/GzipPlugin.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * @copyright 2017 Hostnet B.V.
+ */
+declare(strict_types=1);
+namespace Hostnet\Component\Resolver\Plugin;
+
+use Hostnet\Component\Resolver\Event\FileEvent;
+use Hostnet\Component\Resolver\Event\FileEvents;
+use Hostnet\Component\Resolver\File;
+use Hostnet\Component\Resolver\FileSystem\FileWriter;
+
+/**
+ * This plugin outputs a gzipped file next to the output file. Enabling this with the correct Apache settings
```

Comment:
Enabling this with the correct web server settings

[..]

For example, in Apache this can be done with ...